Python/Tests/TestData/DjangoDebugProject/DjangoDebugProject.pyproj

```diff
@@ -11,7 +11,7 @@
     </SearchPath>
     <WorkingDirectory>.</WorkingDirectory>
     <LaunchProvider>Django launcher</LaunchProvider>
-    <WebBrowserUrl></WebBrowserUrl>
+    <WebBrowserUrl>http://localhost</WebBrowserUrl>
```

Comment:
>http://localhost [](start = 19, length = 16)

Is this change likely to cause the test machines to end up with 1000s of open browser windows?